test/cookbooks/test/recipes/epel.rb

```diff
@@ -1,4 +1,3 @@
-node.default['nginx']['repo_source'] = 'epel'
+apt_update 'update' if platform_family?('debian')
```

Comment:
We shouldn't need this right? 

EPEL should only be used on Redhat derivatives